webapp/webserver.py

```diff
@@ -12,6 +12,10 @@
 # The above copyright notice and this permission notice shall be included in
 # all copies or substantial portions of the Software.
 #
+# Use of the Software is limited solely to applications:
+# (a) running on a Xilinx device, or
+# (b) that interact with a Xilinx device through a bus or interconnect.
+#
```

Comment:
we cannot have this change

Reply:
also your tree is out of sync with xilinx tree, please make sure your tree matches Xilinx tree